First PR for styling OMW

Rohitesh-Kumar-Jain opened the pull request:
Hi In this PR I have: 

- Added a verticle menu on the help page.

- Added a responsive back to the top button which stays with the user as one scroll down.

- Made Images responsive

- Styled the contents box in the OMW. 

- Some small CSS changes such as changing the color of the main nav-bar. 


![Screenshot_2020-11-27 OMW](https://user-images.githubusercontent.com/62026125/100449907-b0ac0780-30da-11eb-85e2-9c6577ec38bb.png)
![Screenshot_2020-11-27 OMW(1)](https://user-images.githubusercontent.com/62026125/100449910-b1dd3480-30da-11eb-8349-c40fa640e8db.png)
![Screenshot_2020-11-27 OMW(2)](https://user-images.githubusercontent.com/62026125/100449915-b3a6f800-30da-11eb-884b-2a6feaa2838c.png)
![Screenshot_2020-11-27 OMW(3)](https://user-images.githubusercontent.com/62026125/100449923-b570bb80-30da-11eb-8462-0f25023e1718.png)
![Screenshot_2020-11-27 OMW(4)](https://user-images.githubusercontent.com/62026125/100449936-b99cd900-30da-11eb-8c66-5cd88944680e.png)
![Screenshot_2020-11-27 OMW(5)](https://user-images.githubusercontent.com/62026125/100449944-bace0600-30da-11eb-9f18-efdb9ebe56b1.png)

- I have kept in mind that styling should be as responsive as possible. 

<img width="1440" alt="Screenshot 2020-11-27 at 6 04 39 PM" src="https://user-images.githubusercontent.com/62026125/100450069-05e81900-30db-11eb-8807-bfe15ae23d0b.png">

Rohitesh-Kumar-Jain committed:
changes to style sheets, styles for images and bordered images added along with a style for command line code.

Rohitesh-Kumar-Jain committed:
images are assigned respective CSS classes

Rohitesh-Kumar-Jain committed:
images are assigned respective CSS classes

Rohitesh-Kumar-Jain committed:
merge conflict fixed

Rohitesh-Kumar-Jain committed:
verticle menu added in the interface with small styling changes in add-wn, feedback_documentation, upload, ili_welcome.

Rohitesh-Kumar-Jain committed:
back to the top button added in long documentations

Rohitesh-Kumar-Jain committed:
stlyed contents in contribute, feedback & feedback_documentation, and contents in search page is modified

Rohitesh-Kumar-Jain commented:
Hi,
There is room for a lot of improvements in the frontend. Post-GSoD I would like to continue working on the frontend (will try to work on the backend as well). I think the styling needs the usage of some Javascript and it's libraries.

Rohitesh-Kumar-Jain committed:
contents table added in ili_welcome template

fcbond commented:
Hi,

it would be great if you can stay and work on the front and back ends.   

For the new styling, I like it.   Can you make the title bar stay even when we scroll up?

And the help menu is not going on the left for me.
![Screenshot from 2020-12-01 11-14-47](https://user-images.githubusercontent.com/1113286/100692822-9c138c00-33c6-11eb-82ab-7be34cfdcf0b.png)

Rohitesh-Kumar-Jain commented:
Hi,

By title bar do you mean the search bar?

Is the help menu supposed to come like this?

<img width="1440" alt="Screenshot 2020-12-01 at 10 04 06 AM" src="https://user-images.githubusercontent.com/62026125/100697791-f0197300-33bc-11eb-8aca-aaa8ccd901fb.png">

fcbond commented:
I did mean the search bar.

I would prefer the left menu to stay underneath the search bar as well.


On Tue, Dec 1, 2020 at 12:48 PM Rohitesh Kumar Jain <
notifications@github.com> wrote:

> Hi,
>
> By title bar do you mean the search bar?
>
> Is the help menu supposed to come like this?
>
> [image: Screenshot 2020-12-01 at 10 04 06 AM]
> <https://user-images.githubusercontent.com/62026125/100697791-f0197300-33bc-11eb-8aca-aaa8ccd901fb.png>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/globalwordnet/OMW/pull/106#issuecomment-736212447>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRX2I5LFHSFNWPOLUODSSRXMBANCNFSM4UE3OW7A>
> .
>


-- 
Francis Bond <http://www3.ntu.edu.sg/home/fcbond/>
Division of Linguistics and Multilingual Studies
Nanyang Technological University

Rohitesh-Kumar-Jain committed:
The search bar now stays on scrolling down

Rohitesh-Kumar-Jain commented:
Hi, team, I have shifted the content bar to the right, and I think if we have a scrolling content section then we don't need back to the top button.
The only disadvantage with the new styling I guess, it doesn't look good on mobiles. 
But I think mostly our users will be using a PC.

Should we stick to the new style? 

I am attaching a link to my screen recording. 

https://drive.google.com/file/d/1pbMWnRrgnPqemq9Yux4qPjZUlQsday7b/view?usp=sharing

Rohitesh-Kumar-Jain committed:
Side navigation menu is embedded in the help page, now interface page and help button are two separate things

Rohitesh-Kumar-Jain committed:
Main help menu on the left, submenu comes in the right, with differnet themes for OMW and CILI

Rohitesh-Kumar-Jain commented:
Hi,
I have made enhancements as per the meeting. 

- Search stays on scrolling.

- Contents bar is on the right and stays on scrolling. 

- On clicking the Help bar, the main navigation comes and it also stays on scrolling, It is on the left side. 

- As we have fixed bars, I have removed back to the top buttons. 

- Separate theming for OMW and CILI is done. 

- Interface documentation and help are now 2 different things.

I have not removed the vertical menu from the interface page, because I feel that I can make it look more elegant in the next PR :sweat_smile:  

Thanks & Regards. 
Rohitesh Jain

fcbond commented:
This looks great, thanks.

fcbond commented:
Sorry, I forgot to confirm the merge.

Rohitesh-Kumar-Jain commented:
No worries : )
